smart-contract-service/src/proto/service.proto

```diff
@@ -214,29 +202,26 @@ message VerifyVyperResponse {
     string abi = 6;
   }
 
-  // `uint32` is used for compatibility purposes, as it transcodes to numeric
-  // value in JSON representation, while `uint64` transcodes to String
-  // (https://developers.google.com/protocol-buffers/docs/proto3#json)
   uint32 status = 1;
   string message = 2;
   optional Result result = 3;
 };
 
-// Is returned as a response for all types of async verification
+// Returned as a response for all types of async verification
 message VerifyAsyncResponse {
-  bytes verification_id = 1; // guid
+  bytes verification_id = 1; // uuid
 };
 
 message GetVerificationStatusRequest {
-  bytes verification_id = 1; // guid
+  bytes verification_id = 1; // uuid
 };
 
 message GetVerificationStatusResponse {
   enum Status {
-    PENDING = 0;
-    PASS = 1;
-    FAIL = 2;
-    UNKNOWN_ID = 3;
+    UNKNOWN_ID = 0;
+    PENDING = 1;
+    PASS = 2;
+    FAIL = 3;
```

Comment:
Life would be much easier that way :upside_down_face:  

But, I'm not sure we could do that here, as current implementation has already established another order for answers (https://github.com/blockscout/blockscout/blob/master/apps/explorer/lib/explorer/chain/smart_contract/verification_status.ex#L62). I'm not sure whether we should or should not be backward compatible in that place, but while we do not know for sure I believe it's better to assume that we should. Otherwise, we may make a system that relies on `UNKNOWN_ID` being the default value and then find out that it is not true anymore